packages/vite-plugin-cloudflare/playground/prisma/package.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "@playground/prisma",
+	"private": true,
+	"type": "module",
+	"scripts": {
+		"build": "vite build --app",
+		"check:types": "tsc --build",
+		"dev": "vite dev",
+		"generate": "pnpm prisma generate",
+		"postinstall": "pnpm generate",
```

Comment:
I had to add this so that the build wouldn't fail when all packages are built by TurboRepo. It might be better for us to omit the playground packages from being built automatically as the builds aren't used elsewhere.

Reply:
Do you think it's a problem that this will run `prisma generate` whenever someone runs `pnpm i` in the repo?

Reply:
mh... yeah it's not ideal 😕 

what about using `predev` and `prebuild` scripts instead?

Something like:
```
"predev": "node ./prisma-setup.mjs",
"prebuild": "node ./prisma-setup.mjs",
```
with `prista-setup.mjs` checking if the prisma is already set up (hopefully there's a simple way to do that?) and running (via child_process) `prisma generate` only when it isn't? (so that on subsequent dev/build runs this script does nothing)

this should avoid the `pnpm i` issue with close to zero overhead when running dev/build

what do you think?

Reply:
[2c2bf54](https://github.com/cloudflare/workers-sdk/pull/8031/commits/2c2bf5449f0a645eea4e15553e022631ffb01823)

I went with just putting `prisma generate` in the `predev` and `prebuild` hooks. It's fast and ensures Prisma is always up to date with the schema.